### ISSUE TITLE
Unable to edit alignment URLs

### ISSUE THREAD

**Issue by siuc-nate** [cassproject/cass-editor#827]
I wanted to fix the alignment URLs that have a / before them, but clicking on them in the edit interface doesn't do anything.
![image](https://user-images.githubusercontent.com/21346519/116271225-67251e80-a745-11eb-9202-a978e4749fc1.png)

Additionally:
- Clicking on the "copy" button triggers the checkmark showing for both URLs, but only the URL that was clicked on is actually copied.
- The first displayed URL in the screenshot above has a `/` before it, but this is not included when that URL is copied via the "copy" button.

**Comment by gloverkari** [comment]
Implemented fix for the copy status indicator to show up on only the copied instance of a property (not on all instances of the property). Additionally improved the input validation for urls. Previous validation looked for http within the url, new validation confirms that the entry starts with http.

**Comment by gloverkari** [comment]
Opened a ticket for the cause of the invalid "/" at the start of the url in the original comment. 
https://github.com/cassproject/cass-editor/issues/1343

**Comment by sedula27** [comment]
The indicator on the selected instance but I am unable to add a new alignment to a competency

**Comment by gloverkari** [comment]
This issue was closed as fixed and then reopened with reference to a different issue. Can we close the original if still fixed and open a new issue if the new behavior on sandbox is still occurring?

**Comment by gloverkari** [comment]
In inability to edit the url is by design. In most cases, the url will be set through an import, a pasted value, or through the selection of another object (via search for...).
The situation that came up in this ticket was due to an issue in the CaSS Editor when a url ends with a slash. I opened a ticket (#1343) on 1/20/24 to track that issue and implemented a fix for it today. The fix for #1343 should avoid the need to manually update the url of any relations.
